spec/index.bs

```diff
@@ -188,6 +188,11 @@ dictionary GPUExtensions {
 <script type=idl>
 dictionary GPULimits {
     u32 maxBindGroups = 4;
+    u32 maxSampledTexturesPerShaderStage = 16;
```

Comment:
Are these meant to be per bindgroup per stage or per pipeline per stage?

Reply:
Makes sense thanks for the explanation.

Reply:
Maybe we can come up with a clearer name here, like PerStagePerBindGroup? verbose though.

Reply:
this semantics indeed needs to be better reflected in the name (unfortunately for that name)